sirepo/pkcli/db.py

```diff
@@ -34,3 +34,86 @@ def _inc(m):
                 break
             with open(str(fn), 'w') as f:
                 f.write(t)
+
+
+def seed_supervisor_state(db_dir, sbatch_poll_secs):
```

Comment:
Agreed on the name.

Should I move it to pkcli/job_supervisor and make a new command in there? I can't move it to sirepo/job_supervisor since the supervisor runs in py3 and this must run in py2.

Reply:
Ah, right. Leave it in db then.